### DISCUSSION TITLE
Is it possible to mount a file from OPFS to WASF using the JS api?

### DISCUSSION THREAD

**Discussion by patrickcorrigan** [emscripten-core/emscripten#21807] [Q&A]
I saw that lots of progress has been made on the JS API for WASMFS https://github.com/emscripten-core/emscripten/issues/15976. I was wondering if it's possible to mount / access a file from OPFS using this api?

**Comment by tlively** [comment]
Yes, I believe `FS.mount(OPFS, {}, '/some-dir');` should do the trick. See https://github.com/emscripten-core/emscripten/blob/main/test/fs/test_mount.c for examples.
